src/components/common/ListNameTranslations.vue

```diff
@@ -116,7 +114,7 @@ export default class ListNameTranslations extends Vue {
 
   .actions {
     .edit-action {
-      border-right: 1px solid $gray2;
+      border-right: 1px solid $gray4;
```

Comment:
Sample screenshot. Note the separator between multiple name translations, and the separator between the Edit button and the dropdown menu arrow.

![image](https://user-images.githubusercontent.com/10002889/202045711-77c91b42-7b7b-47bd-87ae-53d10f2eb14c.png)